.github/actions/maven-build-action/action.yaml

```diff
@@ -0,0 +1,87 @@
+---
+name: "maven-build"
+description: "Setup Maven and run the specified goals"
+
+inputs:
+  jdk-version:
```

Comment:
Anything that isn't required should have a default value (even if it's just an empty string), both to make it clear what will be used if nothing is provided, and as an example of the expected formatting. I think in this case, the defaults that are already in the workflow can be removed from there and put here.

Reply:
Sounds good.. let me update it

Thank you!